manifests/fedora-coreos.yaml

```
@@ -26,6 +26,8 @@ packages:
   - fedora-coreos-pinger
   # Updates
   - zincati
+  # Include and set the default editor
+  - nano-default-editor nano
```

optional (and only because CI failed and has to run again anyway): I prefer this ordering: 

```suggestion
  - nano nano-default-editor
```